### PR TITLE
Resolve failed tests

### PR DESCRIPTION
**Pull request by jeannetoh99**

### CODE REVIEW

File: src/main/java/seedu/address/model/travelplan/UniqueTravelPlanList.java [hunk 1]
```diff
@@ -96,7 +99,7 @@ public void setTravelPlans(List<TravelPlan> travelPlans) {
             throw new DuplicateTravelPlanException();
         }
 
-        internalList.setAll(travelPlans);
+        internalList.setAll(travelPlans.stream().map(tp -> new TravelPlan(tp)).collect(Collectors.toList()));
```
Comment (jeannetoh99) [inline src/main/java/seedu/address/model/travelplan/UniqueTravelPlanList.java:102]: This is to ensure the internalList does not hold the exact same copy of the TravelPlan.

File: src/main/java/seedu/address/model/travelplanner/ModelManager.java [hunk 2]
```diff
@@ -325,9 +325,7 @@ public boolean equals(Object obj) {
         // state check
         ModelManager other = (ModelManager) obj;
         return travelPlanner.equals(other.travelPlanner)
-                && userPrefs.equals(other.userPrefs)
-                && filteredTravelPlans.equals(other.filteredTravelPlans)
-                && filteredWishlist.equals(other.filteredWishlist);
+                && userPrefs.equals(other.userPrefs);
```
Comment (jeannetoh99) [inline src/main/java/seedu/address/model/travelplanner/ModelManager.java:328]: Not too sure if this is okay, removing it resolves execute_validIndexFilteredList_success in DeleteFriendCommandTest

**Review by underthehai — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/model/travelplanner/ModelManager.java (hunk 2 above)
Comment (jeannetoh99) [reply src/main/java/seedu/address/model/travelplanner/ModelManager.java]: This is resolved in the most updated version of this PR.

**Pull request merged**